File: src/main/java/name/zeno/zervice/aop/LogicLog.java
package name.zeno.zervice.aop;

import org.apache.log4j.Logger;
import org.nutz.aop.InterceptorChain;
import org.nutz.ioc.loader.annotation.IocBean;

@SuppressWarnings({"WeakerAccess", "unused"})
@IocBean
public class LogicLog extends MethodLog
{
  Logger logger = Logger.getLogger(LogicLog.class);

  @Override
  public void filter(InterceptorChain chain) throws Throwable
  {
    logger.debug(super.buildMethodArgsReturn(chain));
  }
}

File: src/main/java/name/zeno/zervice/aop/LogicLog.kt
package name.zeno.zervice.aop

import org.apache.log4j.Logger
import org.nutz.aop.InterceptorChain
import org.nutz.ioc.loader.annotation.IocBean

@IocBean
class LogicLog : MethodLog() {
  internal var logger = Logger.getLogger(LogicLog::class.java)

  @Throws(Throwable::class)
  override fun filter(chain: InterceptorChain) {
    logger.debug(super.buildMethodArgsReturn(chain))
  }
}
